app/models.py

```diff
@@ -0,0 +1,16 @@
+from app import db
+
+class Task(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    task_description = db.Column(db.String(64), index=True)
```

Comment:
Two solutions:
- Keep brief and detailed descriptions
- Make this the detailed description